packages/foreman/foreman/foreman.spec

```diff
@@ -711,6 +713,34 @@ Meta Package to install requirements for Redis caching support
 %files redis
 %{_datadir}/%{name}/bundler.d/redis.rb
 
+%package dynflow-sidekiq
```

Comment:
Does this package conflict with the regular dynflow service? Can both run at the same time? What happens if they do? Is the new dynflow-sidekiq optional?

Reply:
No, it doesn't conflict, and it is not trying to remove it, if someone runs both, they will share the load and loadbalance.
`dynflow-sidekiq` is installed in installer only if the option is turned on (theforeman/puppet-foreman#761) so yes, it's optional.

Reply:
Does the singleton nature of the dynflow sidekiq orchestrator play well with the current dynflow executor if both are deployed simultaneously?

Reply:
Yeah, that doesn't really matter, the separation is not really affecting the old deployment.